test/e2e/suites/unmanaged/unmanaged_CAPI_quick_test_cluster_class.go

```diff
@@ -0,0 +1,74 @@
+//go:build e2e
+// +build e2e
+
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package unmanaged
+
+import (
+	"context"
+
+	"github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/utils/pointer"
+
+	"github.com/gofrs/flock"
+	"github.com/onsi/ginkgo/config"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/cluster-api-provider-aws/test/e2e/shared"
+	capi_e2e "sigs.k8s.io/cluster-api/test/e2e"
+)
+
+var _ = ginkgo.Context("[unmanaged] [Cluster API Framework] [smoke] [PR-Blocking]", func() {
+	var (
+		namespace *corev1.Namespace
+		ctx       context.Context
+	)
+
+	ginkgo.BeforeEach(func() {
+		Expect(e2eCtx.Environment.BootstrapClusterProxy).ToNot(BeNil(), "Invalid argument. BootstrapClusterProxy can't be nil")
+		ctx = context.TODO()
+		// Setup a Namespace where to host objects for this spec and create a watcher for the namespace events.
+		namespace = shared.SetupSpecNamespace(ctx, "capi-quick-start", e2eCtx)
+	})
+
+	ginkgo.Describe("Running the quick-start spec with ClusterClass", func() {
```

Comment:
Not sure about Context/Describe texts and I have no idea in which jobs this test is now run (I'm overall not familiar with the CAPA jobs).

Please let me know if something else is more appropriate.

The whole file is just a copy of the other quickstart +/- Describe text and QuickStartSpecInput.Flavor. Maybe it would be better to move the Describe to the other quickstart (file)?

Reply:
I think this is fine.